liquibase-standard/src/main/java/liquibase/logging/mdc/MdcKey.java

```diff
@@ -96,4 +96,14 @@ public class MdcKey {
     public static final String DIFF_CHANGELOG_OUTCOME = "diffChangelogOutcome";
 
     public static final String CHANGESET_SYNC_OUTCOME = "changesetSyncOutcome";
+
```

Comment:
Those ones are going straight to master?

Reply:
My thought was that having the MDC key strings in master wouldn't hurt anything.

Reply:
I backed out those changes.